mobile/src/reducers/calendar/calendar.reducer.ts

```diff
@@ -0,0 +1,30 @@
+import { combineReducers } from 'redux';
+import { combineEpics } from 'redux-observable';
+import { loadDaysEpic$ } from './calendar.epics';
+import { DaysItem, Days } from './days.model';
+import { Reducer } from 'redux';
+import { CalendarActions } from './calendar.action';
+
+export const daysReducer: Reducer<Days> = (state = { vacation: null, off: null, sick: null }, action: CalendarActions) => {
+    switch (action.type) {
+        case 'LOAD-DAYS-FINISHED':
+            return {
+                vacation: action.days.vacation,
+                off: action.days.off,
```

Comment:
don't save bytes. `daysOff`, `vacationDaysLeft`. And what is `sick` supposed to represent? :) Days until sicklist is closed? Then call it appropriately :)

Reply:
Fixed